distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestFederation.java

```diff
@@ -56,7 +58,27 @@
  */
 @RunWith(PaxExam.class)
 @ExamReactorStrategy(PerClass.class)
-public class TestNsili extends AbstractIntegrationTest {
+public class TestFederation extends AbstractIntegrationTest {
```

Comment:
Nearly all of the test classes within Alliance are `TestXXX.java`.  I think we should maintain that convention moving forward.